<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-color: #343434;
}

QLabel {
color: #FFFFFF;
font-size: 12pt;
}

QLineEdit {
background-color: #343434;
font-size: 12pt;
color: #FFFFFF;
}

QPushButton {
font-size: 12pt;
color: #FFFFFF;
background-color: #888888;
}

QComboBox {
background-color: #343434;
font-size: 12pt;
color: #FFFFFF;
}



</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="10" column="0">
    <widget class="QLineEdit" name="register_key_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="re_password_label">
     <property name="text">
      <string>Re-enter password:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="re_password_label_error">
     <property name="styleSheet">
      <string notr="true">color: #FF0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="first_name_label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>First name:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="register_key_label">
     <property name="text">
      <string>Register key recived by post:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="country_label">
     <property name="text">
      <string>Country:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="re_password_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="phone_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="password_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="first_name_label_error">
     <property name="styleSheet">
      <string notr="true">color: #FF0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QComboBox" name="country_comboBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="maxVisibleItems">
      <number>10</number>
     </property>
     <property name="maxCount">
      <number>300</number>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="email_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="phone_label">
     <property name="text">
      <string>Phone number:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="first_name_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="email_label_error">
     <property name="styleSheet">
      <string notr="true">color: #FF0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="last_name_label">
     <property name="text">
      <string>Last name:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="password_label_error">
     <property name="styleSheet">
      <string notr="true">color: #FF0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="last_name_label_error">
     <property name="styleSheet">
      <string notr="true">color: #FF0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="phone_number_label_error">
     <property name="styleSheet">
      <string notr="true">color: #FF0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="last_name_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QPushButton" name="create_account_button">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="password_label">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="email_label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Email:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="register_key_label_error">
     <property name="styleSheet">
      <string notr="true">color: #FF0000;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
